pkgs/by-name/default.nix

```diff
@@ -67,4 +67,14 @@ let
       throw "No package.nix or dream2.nix found in ${directory}"
   ) packageDirectories;
 in
-self
+lib.recursiveUpdate self {
```

Comment:
This looks fishy... why would we add from pkgs here? Ngipkgs is used as an overlay.

Reply:
So the packages would be added to the checks. I'm mostly experimenting with this.